.github/workflows/graviton-build-test-push-workflow-AL2023.yml

```diff
@@ -0,0 +1,309 @@
+name: Graviton Smoke Test WorkFlow AL2023
+on:
+  push:
+    branches:
+      - CSPL_2920
+      - CSPL_xx
+jobs:
+  check-formating:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Dotenv Action
+      id: dotenv
+      uses: falti/dotenv-action@d4d12eaa0e1dd06d5bdc3d7af3bf4c8c93cb5359
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ steps.dotenv.outputs.GO_VERSION }}
+    - name: Check Source formatting
+      run: make fmt && if [[ $? -ne 0 ]]; then false; fi
+    - name: Lint source code
+      run: make vet && if [[ $? -ne 0 ]]; then false; fi
+  #unit-tests:
```

Comment:
what are we planning to do here, are we adding back the test

Reply:
CSPL-2930 - Refactor pipelines(including cleanup)